packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -221,6 +221,15 @@ export interface LaunchTemplateProps {
    */
   readonly launchTemplateName?: string;
 
+  /**
+   * A description for the first version of the launch template.
+   * The version descrioption must be maximum 255 characters long, including hyphens (-),
```

Comment:
A line break would make it clearer.

```suggestion
   * A description for the first version of the launch template.
   * 
   * The version descrioption must be maximum 255 characters long, including hyphens (-),
```